### ISSUE TITLE
Same problem flagged by two rules, hard to disable both

### ISSUE THREAD

**Issue by aheber** [salesforce-ux/stylelint-sds#37]
I have a hard-coded color in a CSS file. This is being flagged by two different rules with the same fix instructions. It seems this should only be reported by a single rule.

```css
.steps {
    background-color: #003674;
}
```

```
Consider replacing the static value for "#003674" with a design token: "
1. --slds-g-color-surface-inverse-1 
2. --slds-g-color-surface-container-inverse-1 
3. --slds-g-color-surface-inverse-2 
4. --slds-g-color-surface-container-inverse-2 
5. --slds-g-color-accent-container-1 
" (slds/no-hardcoded-values-slds2)Stylelint(slds/no-hardcoded-values-slds2)
Consider replacing "#003674" with a similar color "
1. --slds-g-color-surface-inverse-1 
2. --slds-g-color-surface-container-inverse-1 
3. --slds-g-color-accent-container-3 
4. --slds-g-color-surface-inverse-2 
5. --slds-g-color-surface-container-inverse-2 
" styling hook to provide better color contrast (slds/enforce-wcag-rules)Stylelint(slds/enforce-wcag-rules)
```

This is a particular problem because I'm unable to use the recommended fixes from Stylelint to disable both rules.

If I just accept the "disable for the next line" recommendation I end up with something like this, taking the recommendation over and over again.

```
    /* stylelint-disable-next-line slds/no-hardcoded-values-slds2 */
    /* stylelint-disable-next-line slds/enforce-wcag-rules */
    /* stylelint-disable-next-line slds/no-hardcoded-values-slds2 */
    /* stylelint-disable-next-line slds/enforce-wcag-rules */
    /* stylelint-disable-next-line slds/no-hardcoded-values-slds2 */
    background-color: #003674;
```

To properly correct this I need to manually edit the disabling comment

```
    /* stylelint-disable-next-line slds/no-hardcoded-values-slds2, slds/enforce-wcag-rules */
    background-color: #003674;
```

I don't see this plugin as responsible for the recommended fix being insufficient, only that you should know that it happens and other users may get frustrated. I do think the rules should be clear which of them owns this problem and the secondary rule shouldn't report a problem with it, especially when it is effectively the same exactly problem.

**Comment by KishoreNemalipuri** [comment]
We are planning to deleting the `slds/enforce-wcag-rules` rule.

**Comment by aheber** [comment]
Thanks, I can close this knowing it is being taken care of.
